src/arch/Sound/RageSoundDriver_WDMKS.cpp

```diff
@@ -1133,24 +1133,24 @@ void RageSoundDriver_WDMKS::Read( void *pData, int iFrames, int iLastCursorPos,
 		return;
 	}
 
-	int16_t *pBuf = (int16_t *) alloca( iFrames * iChannels * sizeof(int16_t) );
-	this->Mix( (int16_t *) pBuf, iFrames, iLastCursorPos, iCurrentFrame );
+	int16_t pBuf[iFrames * iChannels * sizeof(int16_t)];
```

Comment:
Same problem with setting the array size.  Array size must be a compile time constant.